moped-editor/src/views/projects/projectsListView/test/projectListView.test.js

```diff
@@ -47,7 +47,7 @@ describe("ProjectListView", () => {
 
     // Click the 'Field' dropdown and click an option that uses a lookup table
     await user.click(screen.getByLabelText("Field"));
-    await user.click(screen.getByText("Project type"));
+    await user.click(screen.getByText("Type"));
```

Comment:
This test was failing since the title of this Project List view column change. 🩹

Reply:
🙌